.github/workflows/ci.yml

```
@@ -8,46 +8,61 @@ on:
 jobs:
   lints:
     name: Lints
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-22.04
     steps:
       - uses: actions/checkout@v3
       - uses: cachix/install-nix-action@v17
         with:
           extra_nix_config: |
             access-tokens = github.com=${{ secrets.GITHUB_TOKEN }}
+      - name: Cache lint store (x86_64-linux)
+        id: build-store-x86_64-linux
+        uses: actions/cache@v3
+        with:
+          path: ~/.ci-store
+          key: lint-store-x86_64-linux-${{ hashFiles('**/Cargo.lock', '**/flake.lock') }}-v1
```

Whoops, also missed these.

```suggestion
        id: lint-store-x86_64-linux
        uses: actions/cache@v3
        with:
          path: ~/.ci-store
          key: lint-store-x86_64-linux-${{ hashFiles('**/Cargo.lock', '**/flake.lock') }}-v1
```

---

Oh!!!!